geometry_msgs/Point[] coordinates
---
bool status
